src/screens/shared/farmProfileEdit/FarmProfileEditForm.js

```diff
@@ -22,6 +30,20 @@ const styles = {
   }
 };
 class FarmProfileEditForm extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      farm: {},
+      farmId: ''
+    };
+  }
+
+  async componentDidMount() {
+    const { farmId } = this.props;
+    const farm = await getSingleFarm(farmId);
+    this.setState({ farm, farmId });
```

Comment:
Can you also call `handleChange` on farm in `componentDidMount` to update the parent state and see if it correctly updates state in `FarmProfileEdit`?

`handleChange(farm)`